benchmarks/test_correlation_MG.py

```diff
@@ -18,11 +18,10 @@ def set_up(request):
     logA = 3.05  # log(10^10 A_s)
     cosmo = ccl.Cosmology(Omega_c=0.12/h0**2, Omega_b=0.0221/h0**2, Omega_k=0,
                           h=h0, A_s=np.exp(logA)/10**10, n_s=0.96, Neff=3.046,
-                          m_nu=0.0, w0=-1, wa=0, mu_0=0.1, sigma_0=0.1,
+                          m_nu=0.0, w0=-1, wa=0, T_CMB=2.7255,
```

Comment:
Has @c-d-leonard confirmed this is the value of T_CMB used for this benchmark?

Reply:
I copied it out of the benchmarks that were in C, so yes it is correct.